python/cudf/cudf/core/series.py

```diff
@@ -1811,6 +1812,27 @@ def drop_duplicates(self, keep="first", inplace=False, ignore_index=False):
     def fill(self, fill_value, begin=0, end=-1, inplace=False):
         return self._fill([fill_value], begin, end, inplace)
 
+    def fillna(self, value, method=None, axis=None, inplace=False, limit=None):
+        if isinstance(value, pd.Series):
+            value = Series.from_pandas(value)
+
+        if not (is_scalar(value) or isinstance(value, (abc.Mapping, Series))):
+            raise TypeError(
+                f'"value" parameter must be a scalar, dict '
+                f"or Series, but you passed a "
+                f'"{type(value).__name__}"'
+            )
+
+        if isinstance(value, (dict, Series)):
```

Comment:
Again, we should compare with `abc.Mapping` here

Reply:
Changed this too